about.md

```diff
@@ -8,7 +8,7 @@ title: About
 We've partnered with Dr. Wenhao Sun from [Olakai Hawaii](https://olakai-hawaii.com/) to work on automation of the sea asparagus harvesting process.
 
 ## Team
-The team consists of Max, Joyce, Kevin, Daniel Z, and Daniel Q, all in their 4th year of Mechatronics Engineering at the University of Waterloo.
+The team consists of Max, Joyce, Kevin, Daniel Y, and Daniel Q, all in their 4th year of Mechatronics Engineering at the University of Waterloo.
 
-On top of the mission, *Salico* and projects are designed to meet the requirements of UWaterloo Engineering's [Capstone Design](https://uwaterloo.ca/capstone-design/) project.
```

Comment:
Not really sure what this was trying to say originally

Reply:
just uhh wanted to say that we're doing this partly because its coursework and its tied to UWaterloo

Reply:
Ya I just thought it made more sense to reword it as like "this one project we are working on satisfies capstone"